.github/workflows/python.yaml

```diff
@@ -110,6 +110,28 @@ jobs:
         name: conda-pkgs-${{matrix.os}}
         path: /Users/runner/build
 
+  test_conda_package_install:
+    needs: build_and_test_conda_package
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        image:
+          - ubuntu:16.04
+          - ubuntu:latest
+          - centos:7
+          - centos:latest
```

Comment:
@ddn0 Did I over do the docker image matrix? Should I include anything else? They take about 5min each and all run in parallel after `build_and_test_conda_package` finishes.

Reply:
I might add ubuntu:18.04 and ubuntu:20.04 explicitly (in addition to `ubuntu:latest`). But I don't have a strong preference.